executor/memtable_reader.go

```diff
@@ -893,8 +891,8 @@ func (e *hotRegionsHistoryRetriver) retrieve(ctx context.Context, sctx sessionct
 	}
 	// Merge the results
 	var finalRows [][]types.Datum
-	allSchemas := sctx.GetInfoSchema().(infoschema.InfoSchema).AllSchemas()
 	tz := sctx.GetSessionVars().Location()
+	allSchemas := sctx.GetInfoSchema().(infoschema.InfoSchema).AllSchemas()
```

Comment:
Why change it, how about moving lines `894` and `895` before line `904`?

Reply:
Done.